www.foia.gov/_sass/layouts/_sidebar.scss

```diff
@@ -1,16 +1,30 @@
+.grid-flex {
+  display: flex;
+  justify-content: space-between;
+}
 .sidebar {
+  flex-basis: auto;
+  flex-grow: 5;
+  background-color: $color-primary-darkest; // TODO: Make this $color-primary-alt-lightest when the second tab is active
+  border-bottom: $space-4x solid $color-white;
+  @media screen and (min-width: $medium-screen) {
+    min-width: 420px; //min-width from visual design
+  }
   ul {
     list-style: none;
-    &.sidebar_tab-controls li {
-      display: inline-block;
-      margin-bottom: 0;
-      a {
-        padding: 1rem $space-4x;
+    &.sidebar_tab-controls {
+      margin: 0;
+      li {
         display: inline-block;
-        width: 100%;
-      }
-      &.tab_active a {
-        padding: 1rem 2.8rem; // magic number to keep the tab from changing size when the text is bold
+        margin-bottom: 0;
+        a {
+          padding: 1rem $space-4x;
+          display: inline-block;
+          width: 100%;
+        }
+        &.tab_active a {
+          padding: 1rem 2.8rem; // magic number to keep the tab from changing size when the text is bold
```

Comment:
hope no one is using custom fonts in their browser!

Reply:
We're killing the bold (yay!) so this rule will go away too, just like it deserves.